src/zenml/steps/step_interfaces/base_annotation_step.py

```diff
@@ -0,0 +1,44 @@
+#  Copyright (c) ZenML GmbH 2021. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at:
+#
+#       https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  or implied. See the License for the specific language governing
+#  permissions and limitations under the License.
+"""Base annotation step."""
+
+from abc import abstractmethod
+
+from zenml.steps import BaseStep, BaseStepConfig, StepContext
+
+
+class BaseAlerterStepConfig(BaseStepConfig):
```

Comment:
These are alerter steps but in the base annotation file?

Reply:
Good catch. This should be deleted.